### DISCUSSION TITLE
Amethyst 0.17.0

### DISCUSSION THREAD

**Discussion by ianyh** [ianyh/Amethyst#1393] [Announcements]
* Support for extending existing layouts with custom layouts.
  * Note: this includes a fix for encoding of window frames provided to `getFrameAssignments` that is a potentially breaking change for some custom layouts.
* Support for setting a maximum number of displayed windows. Thanks, @MutatingFunc!
* Support for disabling screen padding on native display. Thanks, @anthdono!
* Support for XDG config hierarchies. Thanks, @cowboy-bebug!
* Fix for not respecting option to not follow windows through space transitions. Thanks, @EvanDorsky!
* Fix for incorrectly considering small windows.
* Fix for an incorrect handling of lack of floating definition.

### Full Changes

* adds rest of layouts to sample config file by @dmartzol in https://github.com/ianyh/Amethyst/pull/1307
* Fix small typo by @max-sixty in https://github.com/ianyh/Amethyst/pull/1312
* Add autominimize setting (AKA window limit) by @MutatingFunc in https://github.com/ianyh/Amethyst/pull/1083
* Support XDG config by @cowboy-bebug in https://github.com/ianyh/Amethyst/pull/1324
* Send console.log in custom layouts to syslog by @rickythefox in https://github.com/ianyh/Amethyst/pull/1322
* Window limit by @MutatingFunc in https://github.com/ianyh/Amethyst/pull/1319
* Issue 1306: Fix "Follow thrown windows between spaces" setting not working by @EvanDorsky in https://github.com/ianyh/Amethyst/pull/1333
* Added option to disable screen padding on built-in display by @anthdono in https://github.com/ianyh/Amethyst/pull/1293
* Always treat quick swapped windows as "tabs" by @ianyh in https://github.com/ianyh/Amethyst/pull/1360
* Fix consideration of small windows by @ianyh in https://github.com/ianyh/Amethyst/pull/1362
* Fix the negative logic of floating list by @ianyh in https://github.com/ianyh/Amethyst/pull/1363
* Allow extending existing layouts by @ianyh in https://github.com/ianyh/Amethyst/pull/1366
* A slew of dependency updates by @ianyh in https://github.com/ianyh/Amethyst/pull/1368
* Update min os version in readme by @ianyh in https://github.com/ianyh/Amethyst/pull/1369

### New Contributors
* @dmartzol made their first contribution in https://github.com/ianyh/Amethyst/pull/1307
* @max-sixty made their first contribution in https://github.com/ianyh/Amethyst/pull/1312
* @MutatingFunc made their first contribution in https://github.com/ianyh/Amethyst/pull/1083
* @cowboy-bebug made their first contribution in https://github.com/ianyh/Amethyst/pull/1324
* @EvanDorsky made their first contribution in https://github.com/ianyh/Amethyst/pull/1333
* @anthdono made their first contribution in https://github.com/ianyh/Amethyst/pull/1293

**Full Changelog**: https://github.com/ianyh/Amethyst/compare/v0.16.1...v0.17.0b

<hr /><em>This discussion was created from the release <a href='https://github.com/ianyh/Amethyst/releases/tag/v0.17.0'>Amethyst 0.17.0</a>.</em>

**Comment by DavidDTA** [comment]
Thank you! I don't yet see this update through the app's "Check for Updates..." option or through `brew upgrade --cask amethyst`. Is there anything else I should do to get the new version, or should I just be patient?

**Reply by ianyh** [reply]
It's not out yet on the update channel, and I don't actually maintain the home brew cask. It just gets magically updated, and I haven't looked at who the saint that updates it is.
